app/models/authorization_request/api_mobilic.rb

```diff
@@ -1,11 +1,22 @@
 class AuthorizationRequest::APIMobilic < AuthorizationRequest
+  def self.basic_infos_attributes
+    %i[intitule]
+  end
+
   include AuthorizationExtensions::BasicInfos
```

Comment:
```suggestion
  add_attribute :intitule
  validates :intitule, presence: true, if: -> { need_complete_validation?(:basic_infos) }
```

et tu peux revert la modification du module, c'est beaucoup plus simple